### ISSUE TITLE
Training from pre-trained model weights.

### ISSUE THREAD

**Issue by Sreerag-ibtl** [qjadud1994/CRNN-Keras#25]
The code uses VGG architecture for feature extraction. My doubt is that, is it possible to begin training from  VGG weights?

**Comment by qjadud1994** [comment]
Initialization does not seem to matter.

Because I had learned the CRNN from scratch.

**Comment by Sreerag-ibtl** [comment]
I tried CRNN with synthetic images of number plates. But the perfomance was poor on real images. So, I was wondering weather it is possible to do transfer learning with a smaller dataset that contains real images.

**Comment by qjadud1994** [comment]
Of course, if you train only with synthetic images, it is natural that it does not work properly in real images.

So, I recommend pre-training with synthetic images and fine-tune with real data.

If you have few real images, it might be worth trying to proceed by learning only the last few layers.

**Comment by Sreerag-ibtl** [comment]
Do you have any suggestion about, upto which layer I should set the trainable flag False. Since there are many different layers involved and GRU layers are also present, I am little confused about this.

**Comment by NikhilShaw** [comment]
Train all layers except pre-trained ones (VGG).

**Comment by centurions** [comment]
> Train all layers except pre-trained ones (VGG).

Can you please explain more ... ? How to do that ? Would you please show example code ? How to freeze those layers in model.py ? 
Thank you
